client-device-connection-infinispan/pom.xml

```diff
@@ -20,20 +20,27 @@
     <version>2.5.0-SNAPSHOT</version>
     <relativePath>../bom</relativePath>
   </parent>
+  <!-- TODO: review in PR request, is there a reason this artifact ID does not start with "hono-"? -->
   <artifactId>client-device-connection-infinispan</artifactId>
 
   <name>Hotrod Device Connection client</name>
   <description>A Hotrod based client for accessing device connection information in an Infinispan data grid.</description>
 
   <dependencies>
     <dependency>
+      <groupId>org.eclipse.hono</groupId>
+      <artifactId>client-device-connection</artifactId>
+    </dependency>
+
+    <dependency> <!-- TODO: should I remove hono-legal and just bring it in transiently from client-device-connection? -->
```

Comment:
@sophokles73 wdyt? Should the legal module dependency always be explicitly declared or can I remove it here and depend on it coming transiently via the `client-device-connection` module?

Reply:
Please keep it. All modules need to explicitly declare this dependency so that we do not mistakenly loose it when fiddling with other dependencies (like you currently do ;-))